repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/SearchCriteriaRepository.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.ccd.definition.store.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.SearchCriteriaEntity;
+
+import java.util.List;
+
+import static uk.gov.hmcts.ccd.definition.store.repository.QueryConstants.SELECT_LATEST_CASE_TYPE_ENTITY_FOR_REFERENCE;
+
+public interface SearchCriteriaRepository extends JpaRepository<SearchCriteriaEntity, Integer> {
+
+    List<SearchCriteriaEntity> findByCaseTypeReferenceIn(List<String> caseTypeReferences);
```

Comment:
same as like this https://github.com/hmcts/ccd-definition-store-api/pull/1061/files#r692719089

Reply:
done in [59c375d1](https://github.com/hmcts/ccd-definition-store-api/pull/1061/commits/59c375d172d81b375f2fe43039955b022afd906e)